### PR TITLE
fixing issues in backup validation + add some missing commands to cli help

### PR DESCRIPTION
**Pull request by foxriver76**
- added commands `fix` and `validate` to help
- fix some issues in backup validation due to the fact, that this.processExit won't stop execution immediately because its calling a callback, thus we have to return to avoid strange errors in stdout
- made some logging in backup validation more meaningful

### CODE REVIEW

**Comment by Apollon77** [comment]
Will merge tomorrow

File: lib/setup/setupBackup.js [hunk 1]
```diff
@@ -649,7 +649,7 @@ class BackupRestore {
     validateBackup(name) {
         return new Promise(resolve => {
             let backups;
-            if (!name && name !== 0) {
+            if (name === undefined || (!name && name !== 0)) {
```
Comment (AlCalzone) [inline lib/setup/setupBackup.js]: I don't understand what this change does.
`!name` is already true for `null`, `undefined`, `""`.

Comment (foxriver76) [reply lib/setup/setupBackup.js]: @Apollon77 added this line while merging and removed it in the next commit :man_shrugging:

Comment (Apollon77) [reply lib/setup/setupBackup.js]: My fault ... I also changed in one of my last commits that stupid ... and fixed it after I saw the real problem in @foxriver76 commit, so I fixed it on both places again

**Pull request merged**